src/Mod/TechDraw/App/DrawViewPart.cpp

```diff
@@ -605,6 +629,22 @@ QRectF DrawViewPart::getRect() const
     return result;
 }
 
+//used to project pt (ex SectionOrigin) onto paper plane
+Base::Vector3d DrawViewPart::projectPoint(Base::Vector3d pt) const

```

Comment:
pt should be passed as const-reference to avoid the creation of a tmp. copy. See also saveParamSpace